test/publish.test.js

```diff
@@ -17,11 +17,12 @@ test('publish (to groupId)', t => {
       recps: [groupId]
     }
     /* NOTE with this we confirm that content.recps isn't mutated while sneakin our own_key in! */
+    // TODO: ssb-tribes/envelope.js has specific logic for if we're in a test or not, we're not using that anymore so this is failing. specifically the msg size. should we maybe just trust that ssb-box2 works correctly/tests this for us?
```

Comment:
@mixmix is this fine? i'd just remove the stuff that has to do with msg size

Reply:
Actions:
- [ ] drop this ENV switcheroo stuff
- [ ] create an issue noting that we're removing envelope sneaky-recps :flamingo:

Reply:
changed the test 9ff3384

Reply:
made issue https://github.com/ssbc/ssb-tribes/issues/101